scripts/hostcfgd

```diff
@@ -402,10 +412,36 @@ class FeatureHandler(object):
         props = dict([line.split("=") for line in stdout.decode().strip().splitlines()])
         return props["UnitFileState"]
 
+    def is_systemctl_status_stable(self, service):
+        """
+        Returns:
+            True: if service status is stable
+            False: if service status is transitioning: Activating, Deactivating
+        """
+        cmd = "sudo systemctl status {}".format(service)
+        process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        stdout, stderr = process.communicate()
+        for line in stdout.strip().split('\n'):
+            if "Active" in line:
+                syslog.syslog(syslog.LOG_INFO, "is_systemctl_status_stable: Active State for feature {} is: {}".format(service, line))
+                if "inactive (dead)" in line or "active (running)" in line: 
```

Comment:
There seems to be other states apart from inactive and active running.
Will it be better to just check if the service is transitioning (activating or deactivating) and return true upon that?